.github/scripts/ci-test-patching.sh

```diff
@@ -16,6 +16,11 @@ declare -a tests_to_skip=(
 
     # This test checks GC logging
     '@test occursin("GC: pause", read(tmppath, String))' "$JULIA_PATH/test/misc.jl"
+
+    # This seems to be a regression from upstream when we merge with upstream 43bf2c8.
+    # The required string int.jl does not appear in the output even if I test with the stock Julia code.
+    # I do not know what is wrong, but at this point, I dont want to spend time on it.
+    '@test occursin("int.jl", code)' "$JULIA_PATH/test/cmdlineargs.jl"
```

Comment:
This is odd, but I guess it might just be a bug in their master. If it doesn't work even with stock Julia, then I agree we shouldn't worry about it.